toolkit/tools/pkg/imagecustomizerlib/customizeusers.go

```diff
@@ -137,5 +137,40 @@ func addOrUpdateUser(user imagecustomizerapi.User, baseConfigPath string, imageC
 		return err
 	}
 
+	// Set user's sudo access.
+	err = UpdateSudoAccess(imageChroot, user.Name, user.Sudo)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func UpdateSudoAccess(installChroot safechroot.ChrootInterface, username string, enableSudo bool) error {
```

Comment:
Was trying to put this function inside userutils.py but will create cyclic dependency issue.